cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -19,8 +19,7 @@
 -->
 {% endif %}
 
-<!--[if lt IE 7]>      <html lang="es" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 7]>         <html lang="en" class="no-js lt-ie9 lt-ie8"> <![endif]-->
```

Comment:
Do we still support IE7?

Reply:
The `lt-ie8` class is still in use in several places so I was hesitant to strip this out, however, perhaps those could be stripped out too.